sgt-back-end/index.js

```diff
@@ -0,0 +1,181 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    SELECT *
+    FROM "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const table = [];
+      for (let i = 0; i < result.rows.length; i++) {
+        table.push(result.rows[i]);
+      }
+      res.json(table);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred'
+      });
+    });
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  const sql = `
+    INSERT INTO "grades" ("name","course","score")
+    values ($1, $2, $3)
+  `;
+  const successfulSql = `
+    SELECT *
+    FROM "grades"
+    WHERE "name"= $1 and "course"= $2 and "score"=$3
+  `;
```

Comment:
If you include a `returning` clause in your `sql`, you can avoid querying the database twice.